lib/publish.js

```diff
@@ -25,7 +25,8 @@ module.exports = async (pluginConfig, context) => {
   const apiOptions = {headers: {'PRIVATE-TOKEN': gitlabToken}};
 
   debug('repoId: %o', repoId);
-  debug('release name: %o', gitTag);
+  debug('release name: %o', name)
```

Comment:
**small issue:** if `name` is falsy, this will output something like:

```
release name: undefined
```

However, the actual network request will be made with `name: gitTag`.

Let's make this log message consistent with what's actually being sent by computing the name above this line and then using the computed name here and in the actual network request:

```js
const releaseName = name || gitTag;

// ...

debug('release name: %o', releaseName)

// ...

json: {
  name: releaseName
}
```